java/change-notes/2020-10-16-guava-flow-steps.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* Some methods of the [Guava](https://guava.dev/) framework have been added as flow steps (specifically those of the [Splitter](https://guava.dev/releases/29.0-jre/api/docs/com/google/common/base/Splitter.html), [Joiner](https://guava.dev/releases/29.0-jre/api/docs/com/google/common/base/Joiner.html), and [Strings](https://guava.dev/releases/29.0-jre/api/docs/com/google/common/base/Strings.html) classes), which may lead to more results from the security queries.
```

Comment:
Guava 30.0 has just been [released](https://github.com/google/guava/releases/tag/v30.0). Would it make sense to adjust the links and the stubs?
Based on the changelog it looks like none of the classes covered by this pull request changed.